client/my-sites/theme/main.jsx

```diff
@@ -475,13 +478,17 @@ const ThemeSheet = React.createClass( {
 
 	renderPrice() {
 		let price = this.props.price;
-		if ( ! this.isLoaded() || this.props.isActive || this.props.isPurchased ) {
+		if ( ! this.isLoaded() || this.props.isActive ) {
 			price = '';
 		} else if ( ! this.props.isPremium ) {
 			price = i18n.translate( 'Free' );
 		}
 
-		return price ? <span className="theme__sheet-action-bar-cost">{ price }</span> : '';
+		const className = classNames( 'theme__sheet-action-bar-cost', {
+			'theme__sheet-action-bar-cost-plan': this.props.price === i18n.translate( 'Plan' )
```

Comment:
Again, I don't see this.

<img width="683" alt="screen shot 2017-06-30 at 11 59 52" src="https://user-images.githubusercontent.com/7767559/27733102-3516a366-5d8c-11e7-8913-2020e83ed7ed.png">